src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestMetadata.cs

```diff
@@ -97,7 +97,7 @@ public IEnumerable<string> Authors
 
         public IEnumerable<string> Owners
         {
-            get { return (_owners == null || !_owners.Any()) ? _authors : _owners; }
+            get { return _owners; }
```

Comment:
Theoretically this is breaking change. Anyone with custom applications using the `NuGet.Packaging` package will have a change in behaviour. However, I think the risk of this breaking someone is very low, so I think it's ok. But we should be aware.